discovery-provider/plugins/notifications/src/processNotifications/mappers/supporterDethroned.ts

```diff
@@ -30,9 +29,9 @@ export class SupporterDethroned extends BaseNotification<SupporterDethronedNotif
   ) {
     super(dnDB, identityDB, notification)
     const userIds: number[] = this.notification.user_ids!
-    this.receiverUserId = this.notification.data.receiver_user_id
-    this.senderUserId = this.notification.data.sender_user_id
-    this.dethronedUserId = this.notification.data.dethroned_user_id
+    this.tipReceiverUserId = this.notification.data.receiver_user_id
+    this.tipSenderUserId = this.notification.data.sender_user_id
```

Comment:
senderUserId doesn't exist anymore and i think i'm even more confused by tipSender and tipReceiver..

the receiver of the notification should be a fellow supporter of the same artist right? not the person receiving a tip

Reply:
yea so the line below on 34 there is a `this.receiverUserId` that is the dethroned user id, and the receiver of the notification.. i could rename that to `notificationRecieverId` if thats more clear?  i tried renaming receiver/sender to tipReceiver and tipSender user ids to clarify that they were not the notification receiver/sender, and it was referring to whoever sent and received the tips that triggered the supporter dethroned notif

wdyt?? should i just change it back?

Reply:
nvm for some reason i thought there was an unassigned variable